weights/VoronoiUtils.cpp

```diff
@@ -27,7 +27,6 @@
 #include <boost/geometry.hpp>
 #include <boost/geometry/geometries/point_xy.hpp>
 #include <boost/geometry/geometries/polygon.hpp>
-#include <boost/geometry/multi/geometries/multi_point.hpp>
```

Comment:
why just remove the include? should `#include <boost/geometry/geometries/multi_point.hpp>` be added here?